tests/_support/Helper/Acceptance/ConvertKitAPI.php

```diff
@@ -9,6 +9,36 @@
  */
 class ConvertKitAPI extends \Codeception\Module
 {
+	/**
+	 * Returns an encoded `state` parameter compatible with OAuth.
+	 *
+	 * @since   2.5.0
+	 *
+	 * @param   string $returnTo   Return URL.
+	 * @param   string $clientID   OAuth Client ID.
+	 * @return  string
+	 */
+	public function apiEncodeState($returnTo, $clientID)
+	{
+		$str = json_encode( // phpcs:ignore WordPress.WP.AlternativeFunctions.json_encode_json_encode
+			array(
+				'return_to' => $returnTo,
+				'client_id' => $clientID,
+			)
+		);
+
+		// Encode to Base64 string.
```

Comment:
I thought there was a library function to do this? I don't know if you're doing this elsewhere